packages/samples/mvp-dapp/src/exception.ts

```diff
@@ -0,0 +1,29 @@
+import { Context, HttpError, Next } from 'koa'
+import { MvpResponse } from './response'
+
+export function exceptionHandle() {
```

Comment:
`exceptionHandler` or `handleException`

Reply:
Okay, I'll refactor it

> `exceptionHandler` or `handleException`

Okay, I'll refactor it